e2e-tests/security-context/run

```diff
@@ -38,12 +38,16 @@ compare_kubectl job.batch/$cluster-xb-$backup
 desc 'run pvc backup'
 compare_kubectl cronjob/each-hour-pvc
 
+pfx=""
+if [[ "$IMAGE_PXC" =~ 8\.0$ ]]; then
+    pfx="-80"
+fi
+
```

Comment:
why it is not part of `compare_kubectl`?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/337/commits/a9f11357452fe547ce7207390a998c985d453cec